sdks/python/test-suites/dataflow/common.gradle

```diff
@@ -458,7 +458,8 @@ project(":sdks:python:test-suites:xlang").ext.xlangTasks.each { taskMetadata ->
         "--runner=TestDataflowRunner",
         "--project=${dataflowProject}",
         "--region=${dataflowRegion}",
-        "--sdk_harness_container_image_overrides=.*java.*,gcr.io/apache-beam-testing/beam-sdk/beam_java8_sdk:latest"
+        "--sdk_harness_container_image_overrides=.*java.*,gcr.io/apache-beam-testing/beam-sdk/beam_java8_sdk:latest",
+        "--sdk_harness_container_image_overrides=.*python.*,gcr.io/apache-beam-testing/beam-sdk/beam_python${project.ext.pythonVersion}_sdk:latest"
```

Comment:
--sdk_container_image for python container?

Reply:
done (both are equivalent in this case)